mage_ai/streaming/sources/rabbitmq.py

```diff
@@ -0,0 +1,98 @@
+from mage_ai.shared.config import BaseConfig
```

Comment:
nit: same here. sort the library in alphabetical order.

Reply:
Fixed on new commit (57b943b2dfdcf4d1ab128bf663facbd64e20beee)